VladKornieiev/src/ACO/week2/homework/library/utils/Utils.java

```diff
@@ -0,0 +1,102 @@
+package ACO.week2.homework.library.utils;
+
+import ACO.data_structures.MyArrayList;
+import ACO.week2.homework.library.controller.interfaces.ILibraryReaderController;
+import ACO.week2.homework.library.model.Library;
+import ACO.week2.homework.library.model.Reader;
+import ACO.week2.homework.library.model.printed_editions.Book;
+import ACO.week2.homework.library.model.printed_editions.Genre;
+import ACO.week2.homework.library.model.printed_editions.Journal;
+import ACO.week2.homework.library.model.printed_editions.PrintedEdition;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.Scanner;
+
+/**
+ * Created by v21k on 31.01.17.
+ */
+public class Utils {
+
+    private static Scanner sc = new Scanner(System.in);
+
+    // to make our library alive
+    public static boolean initLibrary(Library library) {
+        Journal journal = new Journal(Genre.JOURNAL, "Vogue", "Some Author", 2015);
+        Book book = new Book(Genre.FANTASY, "Harry Potter", "Joanne Rowling", 2008);
+        PrintedEdition printedEdition = new PrintedEdition(Genre.SCIENCE, "Science", "MTI", 2013);
+
+        library.addPrintedEdition(journal);
+        library.addPrintedEdition(book);
+        library.addPrintedEdition(printedEdition);
+
+        Reader reader1 = new Reader("Andrii");
+        Reader reader2 = new Reader("Sasha");
+        Reader reader3 = new Reader("Vlad");
+
+        library.addReader(reader1);
+        library.addReader(reader2);
+        library.addReader(reader3);
+
+        return true;
+    }
+
+    public static String showPrintedEditions(MyArrayList al, Comparator comparator) {
+
+        if (al.size() == 0) {
+            return "No printed editions.";
+        }
+
+        PrintedEdition[] printedEditions = printedEditionsToArray(al);
+        Arrays.sort(printedEditions, comparator);
+
+        StringBuilder sb = new StringBuilder();
+        for (int i = 0; i < printedEditions.length; i++) {
+            sb.append(i + 1).append(": ").append(printedEditions[i].toString()).append("\n");
+        }
+        return sb.toString();
+    }
+
+    public static String showPrintedEditions(MyArrayList al) {
+
+        if (al.size() == 0) {
+            return "No printed editions.";
+        }
+
+        StringBuilder sb = new StringBuilder();
+        for (int i = 0; i < al.size(); i++) {
+            sb.append(i + 1).append(": ").append(al.get(i).toString()).append("\n");
+        }
+        return sb.toString();
+    }
+
+    public static int getComparatorChoice() {
```

Comment:
Methods that are responsible for visual part (with Syste.out.print()) should be placed in view part